app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+
+ <div class="field">
+  <%= f.hidden_field :photo, value: 'https://images.unsplash.com/photo-1503023345310-bd7c1de61c7d?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=765&q=80' %>
+  <%= f.hidden_field :bio, value: '' %>
+  <%= f.hidden_field :role, value: 'user' %>
+  <%= f.hidden_field :posts_counter, value: 0 %>
+ <%= f.label :name %><br />
+<%= f.text_field :name, autofocus: true %>
+</div>
+
+
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly ensure your sign up page layout matched the wireframe. The sign up form should be centered and should have increased width that takes up enough space.

  ![image](https://github.com/danutibe07/blogapp/assets/36542986/c971cd94-dd3c-4a60-9126-bd76477e9fdf)